modules/configuration/help/configuration.md

```diff
@@ -0,0 +1,21 @@
+# Configuration
+
+To edit any configuration settings, navigate to the field that you'd
+like to edit in the module, and edit or insert a new value.
+
+Some configuration settings can accept multiple values. For these
+settings, you can add additional fields by pressing the "Add Field"
+button. This will create an empty form area where you can insert
+new values. You can delete any of the settings by pressing the red
+delete button attached to the form.
+
+Press the submit button at the bottom of the page to save your
+changes. You must press the submit button that is on the page where
+you are making the changes for the changes to be stored in the
+database. If you press the submit button on another configuration
+page, it will not store any changes made on other pages.
```

Comment:
The 2 last lines are not clear. Are we talking about change made this page,  another page or other page.

Reply:
Thanks.. I didn't write the content, I just copied it from the old README, but I agree that your wording is more clear, so I updated it.